src/Dashboard/App_Start/NinjectWebCommon.cs

```diff
@@ -1,86 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Diagnostics.CodeAnalysis;
-using System.Linq;
-using System.Web;
-using System.Web.Http;
-using System.Web.Mvc;
-using Dashboard.AppStart;
-using Dashboard.Infrastructure;
-using Microsoft.Web.Infrastructure.DynamicModuleHelper;
-using Ninject;
-using Ninject.Modules;
-using Ninject.Web.Common;
-
-[assembly: WebActivatorEx.PreApplicationStartMethod(typeof(NinjectWebCommon), "Start")]
```

Comment:
After removing this, we're no longer using any of the WebActivator/WebActivatorEx packages right? Can we remove those?